custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_tableaunov17.json

```diff
@@ -12,17 +12,15 @@
     "icds"
   ],
   "config": {
-    "table_id": "static-ccs_record_cases_monthly_tableau_v2",
-    "display_name": "Cases - CCS Record for Tableau v2 (Monthly) (Static)",
+    "table_id": "static-ccs_record_cases_monthly_tableau_nov17",
+    "display_name": "Cases - CCS Record for Tableau Nov 17 (Monthly) (Static)",
     "referenced_doc_type": "CommCareCase",
     "description": "",
     "base_item_expression": {
       "type": "filter_items",
       "items_expression": {
         "expressions": [
-          0,
-          1,
-          2
+          -1
```

Comment:
This works as the months are figured out by doing `iteration -1`. so `-1 -1 = -2` which will be 2 months ago November